src/types.rs

```diff
@@ -699,13 +785,46 @@ impl Term {
             }
         }
     }
-    /// The number of distinct places in the term
+    /// Returns the number of distinct [`Place`]s in the `Term`.
+    ///
+    /// [`Place`]: type.Place.html
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use term_rewriting::{Signature, Term, parse_term};
+    ///
+    /// let mut sig = Signature::default();
+    /// let t = parse_term(&mut sig, "S K").expect("parse of S K");
+    /// let t2 = parse_term(&mut sig, "A(B())").expect("parse of A(B())");
```

Comment:
I think you should be able to write `"A(B)"` here, which might make the test clearer.